android-components/components/support/base/build.gradle

```diff
@@ -67,6 +67,7 @@ dependencies {
     implementation ComponentsDependencies.androidx_lifecycle_viewmodel
 
     api project(":concept-base")
+    implementation project(':ui-widgets')
```

Comment:
Similar comment as [this one](https://github.com/mozilla-mobile/firefox-android/pull/1132#issuecomment-1457192471). Adding this dependency here to a support library has consequences for other components that depend on support libraries too.

Litmus test: does sync-storate and browser-state need to have ui-widgets as a dependency too?

Reply:
From my previous comment, about where to place the change you added to ui-widgets, I didn't realize that it meant adding ui-widgets to the support library. That's my fault for not catching that.